engine/hatchery/kubernetes/kubernetes.go

```diff
@@ -61,6 +61,14 @@ func (h *HatcheryKubernetes) ApplyConfiguration(cfg interface{}) error {
 		}
 	}
 
+	// h.Client = cdsclient.NewService(h.Config.API.HTTP.URL, 60*time.Second)
```

Comment:
Comment are normal, I avoid update to many hatchery for now, waiting branch from @bnjjj 
They will be useful to have hatchery as a CDS Services